pkg/config/tlscfg/testdata/README.md

```diff
@@ -1,35 +1,60 @@
 # Example Certificate Authority and Certificate creation for testing
 
-The following commands were used to create the CA, server and client's certificates and keys
+The following commands were used to create the CA, server and client's certificates and keys. These certificates use the Subject Alternative Name extension rather than the Common Name, which will be unsupported in Go 1.15.
```

Comment:
just curious:
 - couldn't we just move all of this into a script, instead of a run book?
 - is there a flag to for "accept defaults" to make the script even less interactive?

Reply:
Good point, and yes, there's a way to accept defaults. I've modified this README file into a bash script.